README.md

```diff
@@ -21,6 +21,11 @@ Usage
     var map = L.map('map').fitWorld();
     ...
     L.Control.fileLayerLoad({
+        // Allows you to use a customized version of L.geoJson.  
+        // For example if you are using the Proj4Leaflet leaflet plugin, 
+        // you can pass L.Proj.geoJson and load the files into the
+        // L.Proj.GeoJson instead of the L.geoJson.  
+        layer: L.geoJson,
```

Comment:
We should give a class instead because it has to be instantiated with `new L.Proj.GeoJSON()`